docs/supervisor.md

```diff
@@ -14,6 +14,7 @@
 | Velero Plugin for vSphere Version | vSphere Version     | Kubernetes Version                                                | vSphere CSI Driver Version | Velero Version | Velero vSphere Operator Version | Data Manager Version | Velero Plugin for AWS | vSphere Plugin Deprecated | vSphere Plugin EOL Date      |
 |-----------------------------------|---------------------|-------------------------------------------------------------------|----------------------------|----------------|---------------------------------|---------------|----------|------------|---------------|
 | 1.6.0                             | 8.0U3d                  | Bundled with vSphere (1.27-1.29)                                 | Bundled with vSphere       | 1.13.2          | 1.7.0                           | 1.2.0        | 1.6.1 | No         | N/A    |
+| 1.5.4                             | 8.0U3a                  | Bundled with vSphere (1.27-1.29)                                 | Bundled with vSphere       | 1.13.1          | 1.6.1                           | 1.2.0        | 1.6.0 | No         | N/A    |
```

Comment:
Looks like this is identical to the row for 8.0U3 below, can you just change "8.0U3" to "8.0U3/8.0U3a" so that we don't have to add a new row?

Reply:
updated as suggested